modules/templates/SSF/views/layout.html

```diff
@@ -88,6 +88,14 @@
        <li><a href='/{{=appname}}/pr/person/{{=s3_logged_in_person()}}'><i class='icon-edit'></i> Edit Profile</a></li>
        <li><a href='/{{=appname}}/default/user/profile'><i class='icon-user'></i> {{=auth.user.email}}</a></li>
        <li><a href='/{{=appname}}/default/index/subscriptions'><i class='icon-envelope'></i> {{=T("Subscription Settings")}}</a></li>
+       {{if auth.is_logged_in():}}
+         <li>
+           <a href="{{=URL(c="project", f="task", vars={"my_task": "1"})}}">
```

Comment:
Will this work ? I don't think so...
It's okay to keep
/{{=appname}}/project/task?my_task=1
but you can use URL as well, but I doubt this will work because of the use of "